music21/musicxml/xmlToM21.py

```diff
@@ -3177,35 +3161,35 @@ def xmlToDuration(self, mxNote, inputM21=None):
             else:
                 tups = ()
 
-        else: # some rests do not define type, and only define duration
-            durationType = None # no type to get, must use raw
+        else:  # some rests do not define type, and only define duration
+            durationType = None  # no type to get, must use raw
             forceRaw = True
             # TODO: empty-placement
 
         # two ways to create durations, raw (from qLen) and cooked (from type, time-mod, dots)
         if forceRaw:
             if d is not None:
                 # environLocal.printDebug(['forced to use raw duration', durRaw])
-                durRaw = duration.Duration() # raw just uses qLen
+                durRaw = duration.Duration()  # raw just uses qLen
                 # the qLen set here may not be computable, but is not immediately
                 # computed until setting components
                 durRaw.quarterLength = qLen
                 try:
                     d.components = durRaw.components
-                except duration.DurationException: # TODO: Test
-                    qLenRounded = 2.0**round(math.log(qLen, 2)) # math.log2 appears in py3.3
+                except duration.DurationException:  # TODO: Test
+                    qLenRounded = 2.0 ** round(math.log(qLen, 2))  # math.log2 appears in py3.3
                     environLocal.printDebug(['mxToDuration',
-                            'rounding duration to {0} as type is not'.format(qLenRounded) +
-                            'defined and raw quarterlength ' +
-                            '({0}) is not a computable duration'.format(qLen)])
+                                             'rounding duration to {0} as type is not'.format(qLenRounded) +
+                                             'defined and raw quarterlength ' +
+                                             '({0}) is not a computable duration'.format(qLen)])
```

Comment:
These reindentations cause the lines to be too long -- they were unindented purposely to avoid that problem.  :-)

Reply:
I just made a newline before the `[` which means each list element all be aligned and also less indented so it's not too long. Hope that's ok